static/js/components/LearnerInfoCard.js

```diff
@@ -18,50 +21,83 @@ export default class LearnerInfoCard extends React.Component {
     profile: Profile,
     toggleShowPersonalDialog: () => void,
     toggleShowAboutMeDialog: () => void,
+    openLearnerEmailComposer: () => void,
   };
 
-  email = (email: string): React$Element<*> => (
-    <span className="profile-email">{email}</span>
-  );
+  isOwnProfilePage: boolean;
+
+  constructor(props: Object) {
+    super(props);
+    this.isOwnProfilePage = isProfileOfLoggedinUser(props.profile);
```

Comment:
We shouldn't be holding onto props like this. Can we make a function to calculate this value instead?